DataManager/src/DataManagement/ProfileManager.java

```diff
@@ -0,0 +1,91 @@
+package DataManagement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import Utils.NotImplemented;
+
+public class ProfileManager
+{
+	/**
+	 * Organises received data to create a Novus profile in the database.
+	 * <p>
+	 * Splits the string received from the client into the tag, name, languages, and technologies.
+	 * The tag for each section is then dropped and the data is sent to the database interface to be inserted into the database.
+	 * 
+	 * @param in
+	 *            The request that was received from the client
+	 * @return Returns an object list that will only contain a single element; either "added"; or error code 5
+	 * @throws NotImplemented
+	 *             For now, this is not fully implemented, so will throw the NotImplemented exception
+	 */
+	public static List<Object> add ( String in ) throws NotImplemented
```

Comment:
There is two basic pages of documentation for the desktop in the compilation of documentation.